### PR TITLE
<Text /> - added tooltipProps as a new prop

### PR DESCRIPTION
**Pull request by Zivilekin**
### 🔦 Summary

This is second PR of https://github.com/wix/wix-style-react/issues/5668

Things done:
1. Introduce new prop tooltipProps and spread it to Ellipsis component.
2. Add proper types for tooltipProps.

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6261

**Review by mykas — APPROVED** [review]
Super awesome! 🏆

File: src/Text/Text.js [hunk 1]
```diff
@@ -3,12 +3,14 @@ import RawText from './RawText';
 import PropTypes from 'prop-types';
 import { st, classes } from './Text.st.css';
 import Ellipsis, { extractEllipsisProps } from '../common/Ellipsis';
+import { TooltipCommonProps } from '../common/PropTypes/TooltipCommon';
 
-const TextWithEllipsis = ({ className, ...props }) => {
+const TextWithEllipsis = ({ className, tooltipProps, ...props }) => {
```
Comment (mykas) [inline src/Text/Text.js:8]: But why you are not extracting `size` prop form here? it will land on Ellipsis component.

Comment (mykas) [reply src/Text/Text.js:8]: And then we will have a clash

Comment (mykas) [reply src/Text/Text.js:8]: all good, extractEllipsisProps handles that

**Pull request merged**